### DISCUSSION TITLE
Volar and Deno extension conflict

### DISCUSSION THREAD

**Discussion by so1ve** [vuejs/language-tools#946] [Show and tell]
Hey, this is not a problem with this extension. I just want to share this problem and solutions.
If the volar and Deno extensions are installed at the same time, the type inference error will occur:
```
<script setup>
  const foo = ref('bar')
</script>
<template>
  <input v-model="foo" /> // => The type of foo is Ref<"bar">
</template>
```

**Comment by so1ve** [comment]
The workaround is to disable the Deno extension in the workspace.

**Reply by yaegassy** [reply]
Deno extensions are designed to be explicitly enabled and used at the project (workspace) level. For Vue projects, please do not enable Deno extensions.
